wiki/Performance_points/Performance_points_commitee/en.md

```diff
@@ -0,0 +1,21 @@
+---
+stub: true
+---
+
+# Performance Points Committee
+
+The **Performance Points Committee** is a panel consisting of well-versed members of the development community whose main goal is to ensure that changes in the [performance points](/wiki/Beatmap) calculation algorithm could be facilitated, reviewed, and made effective in a fast and steady flow.
```

Comment:
URL here seems off. Performance points have a wiki page of their own.

Reply:
o yeah i copied the wrong url for some reason, whoops